e2e/aws_helpers.go

```diff
@@ -175,20 +177,21 @@ func deleteCertificateAuthority(ctx context.Context, cfg aws.Config, caArn strin
 		panic(deleteErr.Error())
 	}
 
+	delete(testContext.caArns, name)
 }
 
-func createCertificateAuthority(ctx context.Context, cfg aws.Config, isRSA bool) string {
+func (testCtx *TestContext) createCertificateAuthority(ctx context.Context, cfg aws.Config, name string, signingAlgorithm types.SigningAlgorithm) {
```

Comment:
do you want to update the commit message with the changes here too? thanks for updating this method